cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesAliasNamingStrategyFixInvalidIds.java

```diff
@@ -20,24 +20,16 @@
 import java.nio.file.Path;
 import java.util.Arrays;
 import java.util.Map;
-import java.util.Objects;
 import java.util.Optional;
 
 /**
  * @author Miora Vedelago <miora.ralambotiana at rte-france.com>
  */
-public class CgmesAliasNamingStrategy implements NamingStrategy {
+public class CgmesAliasNamingStrategyFixInvalidIds implements NamingStrategy {
```

Comment:
Maybe use a more user-friendly name such as  `FixedCgmesAliasNamingStrategy` and `SimpleCgmesAliasNamingStrategy`?

Reply:
Followed your naming proposal.